### DISCUSSION TITLE
群晖Docker搭建Halo博客+Frp内网穿透+域名解析+SSL证书安装

### DISCUSSION THREAD

**Discussion by jackloves111** [halo-dev/halo#2353] [Show and tell]
针对群晖Docker平台搭建写了一篇教程，供大家参考，有问题可以留言，我再进行优化！
[点击链接查看](https://www.wuzhuwei.work/archives/halojiaocheng)

**Comment by JohnNiang** [comment]
当前链接已经失效，我将关闭当前讨论。
